### PR TITLE
gh-codespace: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 2.2.0

### CODE REVIEW

File: pages/common/gh-codespace.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`gh codespace create`
+
+- List all available codespaces:
+
+`gh codespace list`
+
+- Connect to a codespace via SSH interactively:
+
+`gh codespace ssh`
```
[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/common/gh-codespace.md:16]: But can't you connect to a codespace via HTTPS?

File: pages/common/gh-codespace.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Copy a file to a codespace interactively:
```
Comment (marchersimon) [inline pages/common/gh-codespace.md:18]: Wouldn't `upload` be more fitting here?

File: pages/common/gh-codespace.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/common/gh-codespace.md:16]: This command it's only to open a remote shell, to open the CS in a tab you have the GH's web UI

File: pages/common/gh-codespace.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/gh-codespace.md:18]: We use `transfer` in the `rsync` page.

Comment (marchersimon) [reply pages/common/gh-codespace.md:18]: I'm also fine with that.

**Pull request merged**